services/ResponseCode.proto

```diff
@@ -251,7 +251,7 @@ enum ResponseCodeEnum {
   CUSTOM_FEE_MUST_BE_POSITIVE = 239; // Only positive fees may be assessed at this time
   TOKEN_HAS_NO_FEE_SCHEDULE_KEY = 240; // Fee schedule key is not set on token
   CUSTOM_FEE_OUTSIDE_NUMERIC_RANGE = 241; // A fractional custom fee exceeded the range of a 64-bit signed integer
-  INVALID_CUSTOM_FRACTIONAL_FEES_SUM = 242; // The sum of all custom fractional fees must be strictly less than 1
```

Comment:
If we still keep the `FractionalFee`, I think we should also keep this `INVALID_CUSTOM_FRACTIONAL_FEES_SUM` response code.